libs/external-db-bigquery/src/bigquery_schema_provider.ts

```diff
@@ -14,15 +23,15 @@ class SchemaProvider {
         const res = await this.pool.query(`SELECT table_name, column_name AS field, data_type as type, FROM ${escapeIdentifier(`${this.projectId}.${this.databaseId}`)}.INFORMATION_SCHEMA.COLUMNS`)
         const tables = parseTableData(res[0])
         return Object.entries(tables)
-                        .map(([collectionName, rs]) => ({
+                        .map(([collectionName, rs]: [any, any]) => ({
```

Comment:
recommend to replace it with : 
`  const tables: {[x:string]: {table_name: string, field: string, type: string}[]} = parseTableData( data )`
and then the `collectionName, rs` will be well-defined instead of any.